{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "secalert@redhat.com",
			"ID": "CVE-2013-4260"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "lib/ansible/playbook/__init__.py in Ansible 1.2.x before 1.2.3, when playbook does not run due to an error, allows local users to overwrite arbitrary files via a symlink attack on a retry file with a predictable name in /var/tmp/ansible/."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "CWE-264"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "http://www.ansible.com/security",
					"tags": [
						"Vendor Advisory"
					],
					"url": "http://www.ansible.com/security"
				},
				{
					"name": "http://www.ansible.com/security",
					"tags": [
						"Vendor Advisory"
					],
					"url": "http://www.ansible.com/security"
				},
				{
					"name": "https://bugzilla.redhat.com/show_bug.cgi?id=998227",
					"tags": [
						"Issue Tracking",
						"Patch"
					],
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=998227"
				},
				{
					"name": "https://bugzilla.redhat.com/show_bug.cgi?id=998227",
					"tags": [
						"Issue Tracking",
						"Patch"
					],
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=998227"
				},
				{
					"name": "ansible-cve20134260-symlink(86898)",
					"tags": [
						"VDB Entry",
						"Vendor Advisory"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/86898"
				},
				{
					"name": "ansible-cve20134260-symlink(86898)",
					"tags": [
						"VDB Entry",
						"Vendor Advisory"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/86898"
				},
				{
					"name": "https://groups.google.com/forum/#%21topic/ansible-project/UVDYW0HGcNg",
					"url": "https://groups.google.com/forum/#%21topic/ansible-project/UVDYW0HGcNg"
				},
				{
					"name": "https://groups.google.com/forum/#%21topic/ansible-project/UVDYW0HGcNg",
					"url": "https://groups.google.com/forum/#%21topic/ansible-project/UVDYW0HGcNg"
				}
			]
		}
	},
	"impact": {
		"baseMetricV2": {
			"cvssV2": {
				"accessComplexity": "MEDIUM",
				"accessVector": "LOCAL",
				"authentication": "NONE",
				"availabilityImpact": "PARTIAL",
				"baseScore": 3.3,
				"confidentialityImpact": "NONE",
				"integrityImpact": "PARTIAL",
				"vectorString": "AV:L/AC:M/Au:N/C:N/I:P/A:P",
				"version": "2.0"
			},
			"exploitabilityScore": 3.4,
			"impactScore": 4.9,
			"severity": "LOW"
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:redhat:ansible:1.2.1:*:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:redhat:ansible:1.2:*:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:redhat:ansible:1.2.2:*:*:*:*:*:*:*",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-11-21T01:55Z",
	"publishedDate": "2013-09-16T19:14Z"
}
